operator/cmd/root.go

```diff
@@ -153,6 +153,8 @@ var (
 			// setup operations. This is a hacky workaround until the kvstore is
 			// refactored into a proper cell.
 			identitygc.Cell,
+
+			ciliumendpointslice.Cell,
```

Comment:
Add a short one or two line comment here describing the functionality of this cell. It helps getting an overview of what each top-level cell is doing and how they relate to each other.

Reply:
Done